datadog/metric_test.go

```diff
@@ -37,6 +40,7 @@ import (
 )
 
 func TestMain(m *testing.M) {
+	log.SetOutput(ioutil.Discard)
```

Comment:
I know this is being set to avoid noisy outputs for the benchmark tests, but will it also drop log outputs if a test run fails?

Reply:
Discussed offline: test logs should be unaffected by this because they are logged using a separate `testing` package. However, I have changed the log level to `WarnLevel` so that any logs at warn/error/fatal level remain visible when running benchmark tests.